### PR TITLE
feat : explore and bookmark

### PR DESCRIPTION
**Pull request by anuj-kumary**
PR Include:-
 - Explore  Feed
 - Bookmark Page

### CODE REVIEW

File: src/pages/Bookmark/Bookmark.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { getBookmarkPost } from '../../store/postSlice';
+
+export const Bookmark = () => {
+  const { token, user } = useSelector((state) => state.auth);
+  const { posts } = useSelector((state) => state.posts);
+  const dispatch = useDispatch();
+  console.log(user);
+
+  const getBookmarkPosts = (postId) => posts.filter((post) => post._id === postId)[0];
+  console.log(getBookmarkPosts());
```
Comment (ScoobyDubeyDo) [inline src/pages/Bookmark/Bookmark.js]: **suggestion**:delete console logs if not needed in future

**Review by samyakshah3008 — APPROVED** [review]
Clean code and UI 
Good for approval

File: src/App.js [hunk 2]
```diff
@@ -7,6 +7,8 @@ import Mockman from 'mockman-js';
 import { Navbar } from './components';
 import { Home, Profile, Signin, Signup } from './pages';
 import { EditModal } from './pages';
+import { Bookmark } from './pages/Bookmark/Bookmark';
+import { Explore } from './pages/Explore/Explore';
```
Comment (ScoobyDubeyDo) [inline src/App.js:11]: **suggestion**: barrel export can be used here

**Review by ScoobyDubeyDo — APPROVED** [review]
- the preview looks good
- UI is very nice

File: src/pages/Bookmark/Bookmark.js (hunk 1 above)
Comment (anuj-kumary) [reply src/pages/Bookmark/Bookmark.js]: Done

**Pull request merged**